cfme/tests/containers/test_reports.py

```diff
@@ -91,24 +76,24 @@ def test_container_reports_base_on_options(soft_assert):
         soft_assert(option, 'Could not find option "{}" for base report on.'.format(base_on))
 
 
-@pytest.mark.meta(blockers=[BZ(1435958, forced_streams=["5.8"])])
+@pytest.mark.meta(blockers=[BZ(1467059, forced_streams=["5.8"])])
```

Comment:
If the bug applies to all the tests in the file, you can mark the test at the beginning of the file and apply it to all